src/Simplic.FileStructure.Workflow/Model/DocumentWorkflowContext.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Simplic.FileStructure.Workflow
+{
+    /// <summary>
+    /// Document workflow context to create workflows
```

Comment:
Represents a document workflow configuration. A workflow configuration contains all information about the workflow and how the workflow checks document states, ...
A document workflow might contain multiple documents. Document workflow connections are stored in <ref>